src/styles/charts/line.scss

```diff
@@ -26,6 +26,18 @@ $line-width: 2;
         display: none;
     }
 
+    .custom-line {
+        fill: none;
+        shape-rendering: crispEdges;
+        stroke-width: 1;
+    }
+
+    .custom-line-annotation {
+        font-size: 12px;
```

Comment:
@Golodhros are there any SCSS global vars for this kind of styling?

Reply:
Did not find any, but introduced a new variable and changed the unit to `rem`.